clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3031,10 +3051,14 @@ static void handleMaxGlobalWorkDimAttr(Sema &S, Decl *D,
       return;
     }
   }
-
   if (D->getAttr<SYCLIntelMaxGlobalWorkDimAttr>())
     S.Diag(Attr.getLoc(), diag::warn_duplicate_attribute) << Attr;
 
+  if (checkDeprecatedSYCLAttributeSpelling(S, Attr)) {
+    S.Diag(Attr.getLoc(), diag::note_spelling_suggestion)
+        << "'intel::max_global_work_dim'";
+  }
```

Comment:
Could you please drop curly braces for one statement if conditions?
I see a lot of cases in the patch.

Reply:
Done